### ISSUE TITLE
Add updatable column to project to enable donations

### ISSUE THREAD

**Issue by joshsmith** [code-corps/code-corps-api#261]
https://projects.invisionapp.com/share/RT8PS9FK2#/screens/191325348

The point of this is for the project owner to be able to say:

"Okay, I've got my donation goals set how I want. I'm ready to make this live."

This needs to be explicitly set in the project itself by making an update request, and we need to add a column to do so.

Column could be named something like `donations_enabled`. This _cannot_ be set to `true` unless the project has a connect plan _and_ has donation goals in the project.

**Comment by joshsmith** [comment]
Is this endpoint just creating a `StripeConnectPlan`? Should we set some flag on the project?

We probably _should_ ensure that we do the latter.

**Comment by begedin** [comment]
Feels a bit redundant to do so, IMO. A `donations_enabled` flag would basically be an alias for `stripe_connect_plan_id != nil`. Maybe a computed view property is explicit enough?

**Comment by joshsmith** [comment]
@begedin I don't believe that we render out the plans to everyone, though. So we need to know when to show the donations as enabled on the project page for everyone else, not just for the account itself.

**Comment by begedin** [comment]
Right now, we absolutely do render the plan relationship to anyone (our own id and type), which is enough for the ember app to determine donations are active, but not everyone can fetch the actual plan record, so no important data is exposed.

As I said, it can be added, but it can just as easily be a 

```Elixir
attributes [..., :donations_active, ...]
def donations_active(record, conn) do
  record.plan_id != nil
end
```

On the view, if we want explicitness.

**Comment by joshsmith** [comment]
Okay, I do think I like that explicitness better. Let's take that approach.

**Comment by joshsmith** [comment]
The conversation above misses some things from the UI, now that I'm refamiliarizing myself.

https://projects.invisionapp.com/share/RT8PS9FK2#/screens/191325348

The point of this is for the project owner to be able to say:

> "Okay, I've got my donation goals set how I want. I'm ready to make this live."

This needs to be explicitly set in the project itself by making an update request, and we need to add a column to do so.

**Comment by joshsmith** [comment]
@begedin you're thinking about the case of "when can we display donation goals for a project", which is different from the project owner's "when do I _want_ to accept donations for my project?"

The differences are definitely subtle but wasn't realizing that was what we were discussing until it was made explicit.

**Comment by joshsmith** [comment]
Already partially closed by #494 (and previous PRs).

Technically creating a plan _is_ enough here, but we need to render out what @begedin said above.
